shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -932,7 +932,7 @@ public int touch(TachyonURI path) throws IOException {
    */
   public int unpin(TachyonURI path) throws IOException {
     try {
-      mTfs.setPin(mTfs.open(path), false);
+      mTfs.setPin(mTfs.open(path), true);
```

Comment:
Why is this changed to true? Also I think you can use the `unpin` method which is shorthand for this operation.